core/internal/runhistory/runhistory.go

```diff
@@ -121,9 +115,27 @@ func (rh *RunHistory) Flatten() ([]*service.HistoryItem, error) {
 	return history, nil
 }
 
-// Clone returns a deep copy of the history tree.
-func (rh *RunHistory) Tree() pathtree.TreeData {
-	return rh.pathTree.Tree()
+// GetNumber returns the value of a number-valued metric.
+func (rh *RunHistory) GetNumber(path ...string) (float64, bool) {
```

Comment:
i don't know if want to mess with user provided types, if we are doing it already in the code it's not ideal

Reply:
oh is it only used for testing?

Reply:
yeah, seems like only for testing

JSON numbers are guaranteed to unmarshal to float64 in encoding/json by the way

Reply:
removed it